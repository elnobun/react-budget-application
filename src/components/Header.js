import React from "react";
import { NavLink } from "react-router-dom";

const Header = () => {
  return (
    <div>
      <h1>Budget Manager</h1>
      <NavLink exact to={"/"} activeClassName={"is-active"}>
        Dashboard
      </NavLink>
      <NavLink to={"/create"} activeClassName={"is-active"}>
        AddExpense
      </NavLink>
    </div>
  );
};

export default Header;
