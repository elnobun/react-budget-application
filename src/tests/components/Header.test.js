import React from "react";
import { shallow } from "enzyme";
import Header from "../../components/Header";

test("should render header", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
