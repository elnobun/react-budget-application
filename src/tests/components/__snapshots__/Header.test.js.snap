// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header 1`] = `
<div>
  <h1>
    Budget Manager
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    AddExpense
  </NavLink>
</div>
`;
